collect_app/src/main/res/values/colors.xml

```diff
@@ -20,6 +20,9 @@ the License.
     <color name="green_500">#4CAF50</color>
     <color name="deep_purple_500">#673AB7</color>
 
+    <color name="website_blue">#3e9fcc</color>
```

Comment:
These two new colors look fine in both light and dark modes. I don't really know what they represent yet so given them names based on their colors and putting them here seemed ok.